etc/workflows/fast.yaml

```diff
@@ -34,6 +34,17 @@ operations:
       - overwrite: false
       - accept-no-media: false
 
+  - id: speechtotext
```

Comment:
I know `fast` is supposed to be mainly for testing, but is this going to fail if Whisper/Vosk isn't installed?

Reply:
Indeed, but as I say before, it's only for testing this PR. after approval I will rollback this file and these other files:
- [etc/org.ops4j.pax.logging.cfg](https://github.com/opencast/opencast/pull/4738/files#diff-f815dab0a94901642bd7cd180fcb31d4dcb3ea20585887c575a687fe72f99ca9)
- [etc/org.opencastproject.speechtotext.impl.SpeechToTextServiceImpl.cfg](https://github.com/opencast/opencast/pull/4738/files#diff-44744be62e3ea85de7a6567c32939ad7faefb041c45b7e76b4e0cdbc251c7850)